CHANGELOG.md

```diff
@@ -11,6 +11,10 @@ Every entry has a category for which we use the following visual abbreviations:
 
 ## Unreleased
 
+- 🐞 The port-encoding for Arrow-encoded table slices is now host-independent
+  and always uses network host order.
```

Comment:
```suggestion
- 🐞 The port encoding for Arrow-encoded table slices is now host-independent
  and always uses network-byte order.
```

Reply:
That was the "why don't we have air conditioning in the office"-style changelog entry, dare I say. 🙈